src/flowchem/devices/hamilton/ml600.py

```diff
@@ -90,7 +92,8 @@ async def initialize(self, hw_initialization: bool = True):
         """Ensure connection with pump and initialize it (if hw_initialization is True)."""
         self.num_pump_connected = await self._assign_pump_address()
         if hw_initialization:
-            await self._hw_init()
+            await self.all_hw_init()
+            # initialization take more than 8.5 sec for one instrument
 
```

Comment:
await asyncio.sleep(8) # this might be necessary due to checking request_done sometime fail with "" return

Jakob, do you agree with that?